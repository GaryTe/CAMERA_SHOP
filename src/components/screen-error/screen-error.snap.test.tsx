import {render} from '@testing-library/react';
import {Router} from 'react-router-dom';
import {createMemoryHistory} from 'history';
import ScreenError from './screen-error';

const history = createMemoryHistory();

const children = <ScreenError/>;

describe('Component: ScreenError', () => {
  it('should render ScreenError', () => {

    const {container} = render(
      <Router
        location={history.location}
        navigationType={history.action}
        navigator={history}
      >
        {children}
      </Router>
    );

    expect(container).toMatchSnapshot();
  });
});
