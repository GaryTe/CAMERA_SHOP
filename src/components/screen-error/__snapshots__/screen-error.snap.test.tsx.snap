// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: ScreenError should render ScreenError 1`] = `
<div>
  <div
    class="error"
  >
    <p
      class="error-message"
    >
      Page does not exist
    </p>
  </div>
</div>
`;
